.github/workflows/conda_and_docs_build.yml

```diff
@@ -16,12 +16,13 @@ jobs:
     steps:
     - uses: actions/checkout@v1
     - name: conda-build
-      uses: paskino/conda-package-publish-action@v1.4.3
+      uses: paskino/conda-package-publish-action@v2.0.0
       with:
         subDir: 'recipe'
-        channels: 'conda-forge -c intel -c astra-toolbox/label/dev -c cvxgrp -c ccpi'
+        channels: '-c conda-forge -c intel -c astra-toolbox/label/dev -c cvxgrp -c ccpi'
```

Comment:
channel order is here @paskino

Reply:
so `defaults` it's not there, and I presume it's highest priority if not `--override-channels`?

Reply:
Added --override-channels and still getting the same clobber error